tests/ipc_test.py

```diff
@@ -0,0 +1,245 @@
+from unittest import TestCase, skip
```

Comment:
Does it make sense to test all those IPC methods? Only a couple of them is actually used by Mythril (same for RPC).

Reply:
Actually I don't think it makes sense to test all the IPC methods which not used. But in the challenge, the coverage is required to reach 80%, that means most of the methods needed to be tested.

But I'm OK to remove the unused methods and tests, since the challenge is over :)